response_operations_ui/templates/errors/ru-error.html

```diff
@@ -0,0 +1,16 @@
+{% extends "layouts/base.html" %}
+{% set page_title = "Reporting unit - not loaded" %}
+
+{% block main %}
+    <h1 class="ons-u-mt-m">Reporting unit - not loaded</h1>
+    {% call
+        onsPanel({
+            "type": "error",
+            "classes": "ons-u-mb-l"
+        })
+    %}
+      <p>This reporting unit has been loaded using a sample, but the collection exercise is not yet ready for live. 
+        Please refer to your Survey Processing Centre contact if this is not corrected.</p>
```

Comment:
Should this not be `correct` instead of `corrected`, are we wanting them to do something or just wait until someone else to do something? Should we put a time for them to wait until they should contact someone if we want to keep it like this?

Reply:
This has been corrected. I used the wrong word when writing out the agreed message.

Reply:
Adam M has checked this. I will dismiss this to get it into our branch.